### DISCUSSION TITLE
輸入法使用過程中遇到嘅同音單詞嘅選擇問題001

### DISCUSSION THREAD

**Discussion by liujgoj** [rime/rime-cantonese#146] [Q&A]
| 字詞     | 建議添加嘅音      | 意見 |
|----------|-------------------|------|
| 吖嗎     | aa1 maa3          |   否決，已經確定寫成「吖嘛」   |
| 呀嘛     | aa4 maa5          |    廣州話入面好似冇 aa4 maa5？剩得 aa6 maa5 啊嘛  |
| 十       | aa6               |   呢個係咩嚟嘅？   |
| 唉       | aai2              |   接受，不過改用「哎」   |
| 唉       | aai5              |    接受  |
| 鴨       | aap2              |      |
| 哎       | ai1               |   否決， ai1 唔會單獨成詞   |
| 哎吔     | ai1 yaa1          |  否決，粵拼錯誤    |
| 哎吔     | ai1 yaa3          |   否決，粵拼錯誤   |
| 哎吔     | ai1 yaa4          |   否決，粵拼錯誤   |
| 哎吔     | ai1 yaa5          |    否決，粵拼錯誤  |
| 哎       | ai3               |   否決，ai3 唔會單獨成詞   |
| 哎       | ai5               |   接受，不過改用「唉」   |
| 悒       | ap1               |   否決，「悒」字讀 jap1，至於「心 ap1」點寫待討論   |
| 勾       | au1               |   否決，呢隻字讀 ngau1   |
| 爸       | baa6              |    接受  |
| 叭叭聲   | baa6 baa2 seng1   |   接受   |
| 避       | bei2              |   咩嚟嘅？   |
| 噃       | bo6               |   粵語入面好似冇 bo6 呢個詞？   |
| 𡃓       | bo6               |   同上   |
| 扠       | caa5              |   粵語好似冇 caa5 呢個詞？   |
| 扠堝     | caa5 wo5          |      |
| 賊       | caak2             |   接受   |
| 爭       | caang1            |    呢個係咩嚟嘅？  |
| 陳       | can2              |   否決   |
| 腸       | coeng2            |   接受   |
| 蟲       | cung2             |   否決   |
| 署       | cyu3              |   否決   |
| 大       | daai2             |   否決   |
| 鄧       | dang2             |   否決   |
| 地       | dei2              |      |
| 墩       | deon2             |      |
| 嗒       | dep1              |      |
| D        | di2               |   否決   |
| 地       | di6               |   否決   |
| 定       | ding2             |      |
| 袋       | doi2              |      |
| 涿       | duk1              |      |
| 洞       | dung2             |      |
| 誒       | e2                |      |
| 欸       | e5                |      |
| 欸       | ei2               |      |
| 欸       | ei5               |      |
| 犯       | faan2             |      |
| 份       | fan2              |      |
| 房       | fong2             |      |
| 父       | fu5               |      |
| 𠺢       | gaa1              |   接受   |
| 𠺢嗎     | gaa1 maa3         |   否決，已經有「𠺢嘛」   |
| 𠿪       | gaa2              |   否決，已經有「𠿪」   |
| 嘎       | gaa6              |      |
| 嘎嘛     | gaa6 maa5         |      |
| 𠸏       | ge2               |      |
| 嘰嘰趷趷 | gi6 gi1 gat6 gat6 |   本身就有   |
| 見       | gin2              |   否決   |
| 哥       | go6               |      |
| 吓       | haa2              |   本身就有   |
| 下       | haa5              |   本身就有   |
| 盒       | hap2              |      |
| 𠺪       | he1               |      |
| 𠾍       | hei5              |      |
| 輕       | heng6             |      |
| 哼       | hng1              |      |
| 何       | ho2               |      |
| 意       | i3                |   否決   |
| 要       | iu3               |    否決  |
| 吔       | jaa3              |      |
| 吔       | jaa4              |      |
| 吔       | jaa5              |      |
| 人       | jan2              |      |
| 友       | jau2              |      |
| 油       | jau2              |      |
| 由       | jau5              |      |
| 㖿       | je1               |      |
| 㖿       | je2               |      |
| 咦       | ji5               |      |
| 吆       | jiu2              |      |
| 喲       | jo5               |      |
| 樣       | joeng2            |      |
| 揚       | joeng2            |      |
| 玉       | juk2              |      |
| 凱       | kaai1             |      |
| 拳       | kyun2             |      |
| 嚹       | laa3              |      |
| 喇       | laa4              |      |
| 樓       | lau2              |      |
| 啰       | lo1               |      |
| 啰       | lo1               |      |
| 囉       | lo3               |      |
| 掠       | loek1             |      |
| 囖       | lox               |      |
| 錄       | luk2              |      |
| 亂       | lyun2             |      |
| 吗       | maa1              |      |
| 吗       | maa1              |      |
| 嗎       | maa3              |      |
| 嘛       | maa4              |      |
| 掹       | maang1            |      |
| 雯雯     | man4 man2         |      |
| 茂尼     | mau6 nei2         |      |
| 味       | mei2              |      |
| 面       | min2              |      |
| 毋       | mou2              |      |
| 門       | mun2              |      |
| 匿       | nei1              |      |
| 嗯       | ng1               |      |
| 啞       | ngaa2             |      |
| 嗌       | ngaai3            |      |
| 握       | ngaak1            |      |
| 硬       | ngaangr           |      |
| 鴨       | ngaap2            |      |
| 鴨       | ngaap3            |      |
| 暗       | ngam3             |      |
| 銀       | ngan2             |      |
| 漚       | ngau3             |      |
| 愛       | ngoi3             |      |
| 惡       | ngok3             |      |
| 案       | ngon3             |      |
| 屋       | nguk1             |      |
| 屋企     | nguk1 kei2        |      |
| 㧬       | ngung2            |      |
| 娘       | noeng2            |      |
| 耐       | noi2              |      |
| 哦       | o5                |      |
| 朋       | paang5            |      |
| 譬       | pei5              |      |
| 皮褸     | pei5 lau1         |      |
| 婆       | po2               |      |
| 三       | saa1              |      |
| 嘥       | saai4             |      |
| 思       | si1               |      |
| 繩       | sing2             |      |
| Sir      | soe4              |      |
| 吮       | syun2             |      |
| 太       | taai2             |      |
| 話       | waa2              |      |
| 哇       | waa5              |      |
| 話       | waa5              |      |
| 喂       | wai4              |      |
| 喂       | wai5              |      |
| 堝       | wo5               |      |
| 會       | wui2              |      |
| 喳       | zaa6              |      |
| 姐       | ze1               |      |
| 姐       | ze6               |      |
| 𡀞       | zep1              |      |
| 𡁻       | zeu6              |      |
| 正派     | zing3 paai1       |      |

**Comment by laubonghaudoi** [comment]
麻煩將類似嘅詞彙全部揾晒出嚟，我哋到時一次過解決，唔該晒你

**Comment by liujgoj** [comment]
好嘅。等我將電影檢查完。一次性提交畀你。

**Comment by liujgoj** [comment]
[字词库反饋.xls](https://github.com/rime/rime-cantonese/files/7939712/default.xls)
你好，我而家將做電影字幕過程中遇到嘅漏音嘅字詞全部列喺EXCEL入邊，供你哋參考。

**Reply by laubonghaudoi** [reply]
我啱先幫你改咗個討論帖，你睇個表格，以後直接喺個表格入面加就得
